UNRELEASED.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 > Place your changes below this line.
+**Fixed:**
+ - The range-selection colour in the calendar has been toned down a bit.
```

Comment:
This is very subject to interpretation, maybe it could be a bit more literal and say what's changed (e.g. the track colour has changed from oldColor to newColor)

Reply:
```suggestion
 - The range track background colour is now less vibrant than the start/end dates background colour.
```